zone/npc.cpp

```diff
@@ -3573,8 +3573,8 @@ void NPC::AIYellForHelp(Mob *sender, Mob *attacker)
 
 	LogAIYellForHelp(
 		"NPC [{}] ID [{}] is starting to scan",
-		GetCleanName(),
-		GetID()
+		(!this ? "NULL MOB" : GetCleanName()),
```

Comment:
This change shouldn't be necessary. `this` can never be null in well formed code, so something else has to be the cause of the crash if it's crashing here (or maybe the line numbers were wrong?)


(checking `this` against null I guess was a thing back in the 90s, but that's because the compilers/implementations were special ...)

Reply:
Okay, that makes sense. not sure what else it would have been there. like you said, maybe the line numbers are different now, since this crash was last week.